Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -337,11 +340,12 @@ static inline void GenerateVertexShader(T& out, u32 components, API_TYPE api_typ
 	}
 
 	// clipPos/w needs to be done in pixel shader, not here
-	out.Write("o.clipPos = float4(pos.x,pos.y,o.pos.z,o.pos.w);\n");
+	out.Write("o.clipPos = float4(0.0, 0.0, o.pos.z, o.pos.w);\n");
```

Comment:
We might as well just do `o.clipPos = o.pos` here. I still wouldn't adjust it in the stereoscopy code since we don't use it though. Might actually be useful in the pixel shader to have the monoscopic position in the pixel shader stage.